### ISSUE TITLE
Desktop Internet Explorer Touch Screen

### ISSUE THREAD

**Issue by MarcelVe** [danniehansen/Touch-box#6]
Hi, can you help?
I need that this touchbox to work in internet explorer on desktop windows 7 or superior.
Sorry, my english is not so good.
Thank you.

**Comment by danniehansen** [comment]
Hello Marcel,

Which internet explorer version are you looking for it to work? & is it IE touch that you are having issues with?

**Comment by MarcelVe** [comment]
Yes, Internet Explorer 9 or superior with a touch screen monitor.

**Comment by danniehansen** [comment]
Alright, i will look into this now. But i will need for you to test it once I'm done if you can =)

**Comment by MarcelVe** [comment]
Ok, thank you very much.

**Comment by danniehansen** [comment]
There we go, new fresh commit with several new changes & fixes. Could you give it a shot on IE touch?

**Comment by MarcelVe** [comment]
Yes, where can i download it?

**Comment by danniehansen** [comment]
It's on github. I commited it up.

**Comment by MarcelVe** [comment]
Ok, i will test it. Thanks.

**Comment by MarcelVe** [comment]
As soon as possible i will feedback you. Thank you.

**Comment by MarcelVe** [comment]
Hi, this plugin still not working in windows 7 ie9 and higher and windows 8 ie10 and higher.
I hope you can help me.
Thank you.

**Comment by danniehansen** [comment]
@MarcelVe Do you have any test suit that support IE9 touch screen monitor? Reason why I'm asking is that i do not have any touch screen monitor running IE9. All i have is a few tablets & www.browserstack.com - so until we figure out a way for me to test this i cannot do much.
